br/pkg/lightning/restore/meta_manager.go

```diff
@@ -267,7 +267,7 @@ func (m *dbTableMetaMgr) AllocTableRowIDs(ctx context.Context, rawRowIDMax int64
 				found := false
 				for _, info := range autoIDInfos {
 					if strings.ToLower(info.Column) == autoIDField {
-						maxRowIDMax = info.NextID - 1
+						maxRowIDMax = int64(info.NextID) - 1
```

Comment:
this should also be changed to uint64 rather than cast it to int64. otherwise we can't handle the large values like 10942694589135710585. But I think many old codes already assume the rowID type is int64 and I'm not sure we can change them 🤔 Will take a look later